modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -65,23 +65,27 @@ $ oc adm upgrade --to-multi-arch
 $ oc adm upgrade
 ----
 +
+.Example output
+
+[source,terminal]
+----
+working towards ${VERSION}: 106 of 841 done (12% complete), waiting on machine-config
+----
++
 [IMPORTANT]
 ====
 Machine launches may fail as the cluster settles into the new state. To notice and recover when machines fail to launch, we recommend deploying machine health checks. For more information about machine health checks and how to deploy them, see _About machine health checks_.
 ====
 +
-//commenting this section out until https://issues.redhat.com/browse/OCPBUGS-8256 is resolved:
-//For `oc get co`, expect `AVAILABLE=True`, `PROGRESSING=False`, and `DEGRADED=False` on all cluster Operators.
-+
-//For `oc get mcp`, expect `UPDATED=True`, `UPDATING=False`, and `DEGRADED=False` on all machine config pools.
+* If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:
```

Comment:
I feel like if you are introducing another step in this verification section, both steps now need to be numbered list items:
```suggestion
. If you want more detailed information about the status of your update, you can also monitor the migration by running the following Technology Preview command:
```